liqd_lunch/frontend/src/ResistanceList.js

```diff
@@ -38,28 +46,58 @@ class  ResistanceList  extends  Component {
       )
     }
 
+    timeForLunch() {
+      var now = new Date();
+      if (now.getHours() >= 12) {
```

Comment:
change here to test e.g. to 
if (now.getSeconds() >= 30)
then it switch every 30 seconds